extensions/BugModal/template/en/default/bug_modal/edit.html.tmpl

```diff
@@ -298,7 +298,7 @@
         <button type="button" id="mode-btn" class="major">
           <span id="mode-btn-readonly" title="Enable editing fields for [% terms.bug %] metadata">Edit [% terms.Bug %]</span>
           <span id="mode-btn-loading">
-            <img id="edit-throbber" src="extensions/BugModal/web/throbber.gif" width="16" height="11">
+            <img id="edit-throbber" src="[% asset_file('extensions/BugModal/web/throbber.gif') FILTER html %]" width="16" height="11">
```

Comment:
I read this as '.gif FILTER html' and wodnered why we are emitting a .gif reinterpreted as HTML. What does this actually do in practical HTML output?